spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Index View Page', type: :system do
```

Comment:
- [ ] Please make sure that your `type` should be `feature` not the `system`. The difference is that you will get a `headless` test without opening the chromium window. Kindly do it in your all feature test files.